src/components/forms/set-password-form.tsx

```diff
@@ -30,9 +30,9 @@ const SetPasswordForm: React.FC<{
   const { formatMessage } = useIntl();
 
   return (
-    <StyledForm
+    <Form
       onSubmit={async (event) => {
-        event.preventDefault();
+        event && event.preventDefault();
 
```

Comment:
Can `event` be undefined? 🤔

Reply:
I think not, but at some places we don't pass it as an argument to the callback so 🤷‍♀️